packages/core/src/App/Containers/Modals/information-submitted-modal/information-submitted-modal.scss

```diff
@@ -0,0 +1,10 @@
+.information-submitted-modal {
+    @include mobile {
+        .dc-dialog {
+            &__content,
+            &__button {
+                font-size: 1.2rem;
```

Comment:
is there a chance to adjust Text component with font size right in jsx file?

Reply:
This file is removed